pkg/volume/volume_linux_test.go

```diff
@@ -342,6 +342,11 @@ func TestProgressTracking(t *testing.T) {
 		time.Sleep(permissionSleepDuration)
 		return nil
 	}
+	defer func() {
```

Comment:
can this capture existing values and reset to existing values rather than duplicate the assignments? hard-coding here means if we change the package defaults these will modify values

```go
func TestProgressTracking(t *testing.T) {
  originalFilePermissionChangeFunc = filePermissionChangeFunc
  originalProgressReportDuration = progressReportDuration
  originalFirstEventReportDuration = firstEventReportDuration
  t.Cleanup(func(){
    filePermissionChangeFunc = originalFilePermissionChangeFunc
    progressReportDuration = originalProgressReportDuration
    firstEventReportDuration = originalFirstEventReportDuration
  })
  ...
```

Reply:
good idea. fixed.